homeassistant/components/netgear/__init__.py

```diff
@@ -52,14 +55,33 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         configuration_url=f"http://{entry.data[CONF_HOST]}/",
     )
 
+    async def async_update_data():
+        """Fetch data from the router."""
+        await router.async_update_device_trackers()
+        return
+
+    # Create update coordinator
+    coordinator = DataUpdateCoordinator(
+        hass,
+        _LOGGER,
+        name=router.device_name,
+        update_method=async_update_data,
+        update_interval=SCAN_INTERVAL,
+    )
+
```

Comment:
The documentation suggests you call `await coordinator.async_config_entry_first_refresh()` when the coordinator has been setup.

https://developers.home-assistant.io/docs/integration_fetching_data/